tests/session/cmd/test_legacy.py

```diff
@@ -89,3 +89,29 @@ def test_legacy_run_sequential(tox_project: ToxProjectCreator, mocker: MockerFix
     tox_project({"tox.ini": ""}).run("le", "-e", "py")
 
     assert run_sequential.call_count == 1
+
+
+def test_ensure_all_environments_are_executed(tox_project: ToxProjectCreator) -> None:
+    ini = """
+    [tox]
+    env_list = aaa
+    [testenv]
+    [testenv:bbb]
+    description = not in env_list
+    """
+    setup_py = """
```

Comment:
You don't neeed the package part, the problem is independent weather there's a packaging env or not. And when that's a true is better to not have it to make the test faster.

Reply:
Nice catch and thanks for the explanation!